mods/cnc/maps/nod02a/map.yaml

```diff
@@ -0,0 +1,450 @@
+MapFormat: 7
+
+RequiresMod: cnc
+
+Title: Invasion of Egypt (a)
+
+Description: GDI has kept a stranglehold on Egypt for many years.  Set up a forward attack base in your area.  To do this you must double click on your Mobile Construction Vehicle.  (MCV)  From here you can begin to build a base. This area contains plenty of Tiberium, so establishing the base should be easy.
+
+Author: Westwood Studios
+
+Tileset: DESERT
+
+MapSize: 64,64
+
+Bounds: 27,34,36,26
+
+Visibility: MissionSelector
+
+Type: Campaign
+
+Videos:
+	Briefing: nod2.vqa
+	GameLost: deskill.vqa
+	GameWon: airstrik.vqa
```

Comment:
The correct file is `airstrk.vqa` as they only had 8 characters in the good old DOS days.